### PR TITLE
SES 133 improvements 6 add a plugin to eslint to correct spelling problems

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
133-improvements-6-add-a-plugin-to-eslint-to-correct-spelling-problems-v-20

# Description

# Actions
Add regex and and word to eslintrcjs
fix-problem-core-uni-summary
Add eslint-plugin-spellcheck

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -11,16 +12,120 @@ module.exports = {
...
+          'Hight',
+          'Subheader',
+          'Identicon',
+          'ident',
+          'Unordered',
+          'juan',
+          'Julien',
+          'wouter',
+          'Kampmann',
+          'Wouter',
```
Comment (z3phyro) [inline .eslintrc.js:105]: Do you think these are needed on the skip words dictionary?

File: .eslintrc.js [hunk 2]
```diff
@@ -11,16 +12,120 @@ module.exports = {
...
+          'Tooltip',
+          'Team’s',
+          'Petru',
+          'Catana',
+          'Jevans',
```
Comment (z3phyro) [inline .eslintrc.js:110]: Do you think these are needed on the skip words array?

File: .eslintrc.js [hunk 3]
```diff
@@ -11,16 +12,120 @@ module.exports = {
...
+          'onchain',
+          'Whitepaper',
+          'Onboarding',
+          'gtag',
+          'mailto',
+          'Millis',
+          'Datetime',
+          'Sitemap',
+          'Mip', // Explicity word to talk about mips
+        ],
```
Comment (z3phyro) [inline .eslintrc.js:120]: Can we move this array to a different file .json to make the config more readable?

Comment (alejandrocabriales) [reply .eslintrc.js]: you are right, I will move to array

**Pull request merged**